.github/workflows/release-canary.yml

```diff
@@ -0,0 +1,29 @@
+name: Release canary package
+on:
+  push:
+    branches:
+      - v2-develop
+  workflow_call:
+    inputs:
+      ENVIRONMENT:
+        required: true
+        type: string
```

Comment:
Do we need a workflow_call here? I though we wanted it only for RC releases?

Reply:
I changed during your review now its on 
```
on:
  push:
    branches:
      - v2-develop
      ```